### PR TITLE
chore(): regression fix for `4.0.0-beta10-ocean`

### PR DESCRIPTION
**Pull request by canonbrother**
#### What this PR does / why we need it:

as per title

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

**Comment by canonbrother** [comment]
own note: stackModifier missing in block 0 🤔 
https://github.com/BirthdayResearch/jellyfishsdk/blob/9ba97250a1ec5ff29642cd5420e110b20a207ffb/apps/whale-api/src/module.indexer/model/block.e2e.ts#L31

File: packages/jellyfish-transaction-builder/__tests__/txn/varint_gt_128.test.ts [hunk 1]
```diff
@@ -599,56 +599,58 @@ it('should setCollateralToken, takeLoan and paybackLoan', async () => {
...
+  //   const col = await testing.rpc.loan.getCollateralToken('FISH#128')
+  //   expect(col).toStrictEqual({
+  //     token: 'FISH#128',
+  //     tokenId: expect.stringMatching(/[0-f]{64}/),
+  //     factor: new BigNumber(1),
+  //     fixedIntervalPriceId: 'FISH#128/USD'
+  //   })
+  // }
+
+  // not allowed for non-DAT token
```
Comment (fuxingloh) [inline packages/jellyfish-transaction-builder/__tests__/txn/varint_gt_128.test.ts:627]: What happened to this?

**Review by fuxingloh — APPROVED** [review]
LGTM

> On another note, are we doing token naming migration?

File: packages/jellyfish-transaction-builder/__tests__/txn/varint_gt_128.test.ts (hunk 1 above)
Comment (canonbrother) [reply packages/jellyfish-transaction-builder/__tests__/txn/varint_gt_128.test.ts:627]: https://github.com/DeFiCh/ain/blob/ea7b1806babc3af760df30b136b34c996d67b29f/src/masternodes/govvariables/attributes.cpp#L1776-L1784

hmm.. its prev bug.. as non DATs should not be set as collateral tokens in the native DEX

fixes at:
https://github.com/DeFiCh/ain/pull/2258#discussion_r1287555257

**Comment by canonbrother** [comment]
> On another note, are we doing token naming migration?

the name refers the same as origin
eg:
https://github.com/DeFiCh/ain/blob/322e3398c6f8c3a81c965314c406d0c799d004ba/test/functional/feature_dst20.py#L271-L294

**Review by pierregee — APPROVED** [review]

**Review by thedoublejay — APPROVED** [review]

**Pull request merged**